src/app/services/tests/AuthenticationService.spec.ts

```diff
@@ -15,30 +15,36 @@ import * as FireAuth from '@angular/fire/auth';
 import { ConnectivityDAO } from 'src/app/dao/ConnectivityDAO';
 import { ErrorLoggerService } from '../ErrorLoggerService';
 import { ErrorLoggerServiceMock } from './ErrorLoggerServiceMock.spec';
+import { User } from 'src/app/domain/User';
 
 @Injectable()
 export class AuthenticationServiceMock {
-    public static CONNECTED_UNVERIFIED: AuthUser = new AuthUser(MGPOptional.of('jean@jaja.europe'),
+    public static CONNECTED_UNVERIFIED: AuthUser = new AuthUser('jeanjaja123',
```

Comment:
real debate: AuthUser is from AuthenticationServiceMock or for UserMocks :D ?

Reply:
AuthUser = utilisateur loggé, donc pour AuthenticationService. Après ptet y'a moyen de ticketter le merge des deux ?

Reply:
J'serais d'avis de centraliser les données similaires ensemble, pas utiliser du creator/opponent/observer dans un fichier et du jean-java/laponira/bitekuteku dans AuthUserMock :p

Reply:
mais creator/opponent/observer ça n'a pas de sens pour tester l'auth service? Et CONNECTED_VERIFIED, CONNECTED_UNVERIFIED n'ont pas de sens dans les jeux en ligne non plus.

Reply:
Certes, donc je propose qu'on ai dans un seul mock de fichier, centralisé, ces différents users, leurs nom regardant des tests différents mais leur type de données est user-related pour tous.

Reply:
tout bougé dans UserMocks donc !